src/typings.d.ts

```diff
@@ -1,5 +1,5 @@
 /* SystemJS module definition */
-declare var module: NodeModule;
+declare var module: NodeJS.Module;
```

Comment:
@YouQam What has happened here?

Reply:
The PR build was failing due to some type conflicts. This update resolve the issue.